src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_constants.py

```diff
@@ -14,3 +14,5 @@
 
 CLASSIC_REGISTRY_SKU = ['Classic']
 MANAGED_REGISTRY_SKU = ['Basic', 'Standard', 'Premium']
+
+MODE = ['Force', 'Noforce']
```

Comment:
What do other modules use? I know that PowerShell uses a confirm flag which is native to cmdlets? 
Can we check if any other modules have a confirmation flag.. Should we just have --overwrite for import?